tutorials/3d/reflection_probes.rst

```diff
@@ -107,4 +107,6 @@ In the current renderer implementation, all probes are the same size and
 are fit into a Reflection Atlas. The size and amount of probes can be
 customized in Project Settings -> Quality -> Reflections
 
+The default settings of Atlas Size: 2048 and Atlas Subdiv: 8 will allow up to 16 reflection probes in a scene. These values need to be increased if you need more.
```

Comment:
Atlas size doesnt actually affect how many probes you can have in the scene, it just affects their resolution. 

```suggestion
The default setting of Atlas Subdiv: 8 will allow up to 16 reflection probes in a scene. This value needs to be increased if you need more reflection probes.
```